tests/conftest.py

```diff
@@ -22,3 +22,8 @@ def temppath(tmpdir):
 @pytest.fixture
 def plugin(temppath) -> dummyplugin.DummyPlugin:
     return dummyplugin.DummyPlugin(temppath)
+
+
+@pytest.fixture
+def assets_dir() -> pathlib.Path:
```

Comment:
I think we should have a `moodle_assets_dir` fixture in `test_moodle.py` which adds a `moodle` subfolder to this - right now it's quite a chaos in `tests/assets`.